content/ja/docs/reference/kubectl/jsonpath.md

```diff
@@ -0,0 +1,112 @@
+---
+title: JSONPathのサポート
+content_type: concept
+weight: 25
+---
+
+<!-- overview -->
+KubectlはJSONPathのテンプレートをサポートしています。
+
+<!-- body -->
+
+JSONPathのテンプレートは、中括弧({})によって囲まれたJSONPathの式によって構成されています。
+Kubectlでは、JSONPathの式を使うことで、JSONオブジェクトの特定のフィールドをフィルターしたり、出力のフォーマットを変更することができます。
+本来のJSONPathのテンプレートの構文に加え、以下の機能と構文が使えます:
+
+1. JSONPathの式の内部でテキストをクォートするために、ダブルクォーテーションが使えます。
+2. リストを反復するために、`range`、`end`オペレーターが使えます。
+3. リストを末尾側から参照するために、負の数のインデックスが使えます。負の数のインデックスはリストを「周回」せず、`-index + listLength >= 0`が満たされる限りにおいて有効になります。
+
+{{< note >}}
+
+- 式は常にルートのオブジェクトから始まるので、`$`オペレーターの入力は任意になります。
+
+- 結果のオブジェクトはString()関数を適用した形で表示されます。
+
+{{< /note >}}
+
+以下のようなJSONの入力が与えられたとします。
+
+```json
+{
+  "kind": "List",
+  "items":[
+    {
+      "kind":"None",
+      "metadata":{"name":"127.0.0.1"},
+      "status":{
+        "capacity":{"cpu":"4"},
+        "addresses":[{"type": "LegacyHostIP", "address":"127.0.0.1"}]
+      }
+    },
+    {
+      "kind":"None",
+      "metadata":{"name":"127.0.0.2"},
+      "status":{
+        "capacity":{"cpu":"8"},
+        "addresses":[
+          {"type": "LegacyHostIP", "address":"127.0.0.2"},
+          {"type": "another", "address":"127.0.0.3"}
+        ]
+      }
+    }
+  ],
+  "users":[
+    {
+      "name": "myself",
+      "user": {}
+    },
+    {
+      "name": "e2e",
+      "user": {"username": "admin", "password": "secret"}
+    }
+  ]
+}
+```
+
+Function            | Description               | Example                                                         | Result
```

Comment:
I would also translate these headings.

Reply:
Thanks for the sugestion!

Translated in 8e9b1564a9f540644863ed6ed05450744e5fd711